ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6618,14 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+PSM:
```

Comment:
Can `WDPM` be used here? If `differential_pressure_sensor` is missing then I would flag it as missing for your unit rather than creating a new type for the alarms/status only as the alarm logic likely depends on this sensor.